src/ray/core_worker/core_worker.cc

```diff
@@ -2208,11 +2208,11 @@ void CoreWorker::HandlePushTask(const rpc::PushTaskRequest &request,
   // execution service.
   if (request.task_spec().type() == TaskType::ACTOR_TASK) {
     task_execution_service_.post(
-        [=] {
+        [this, request, reply, callback = std::move(send_reply_callback)] {
```

Comment:
```suggestion
        [this, request, reply, send_reply_callback = std::move(send_reply_callback)] {
```

This is much clear to me.